pontoon/base/urls.py

```diff
@@ -121,6 +122,12 @@
         TemplateView.as_view(template_name='terms.html'),
         name='pontoon.terms'),
 
+    # Terms
+    url(r'^persona-migration/$',
+        login_required(
+            TemplateView.as_view(template_name='persona-migration.html'),
+        ), name='pontoon.persona-migration'),
+
```

Comment:
Shuld we move this to the main `urls.py` to keep all authentication-oriented URLs in one place?